.gitignore

```diff
@@ -1,5 +1,6 @@
 # testing configs
-for-testing
+output
+config
```

Comment:
For `config`, I wasn't sure but yeah it makes sense to put some default configs in. As for `output`, the directory gets automatically created when Docker mounts the volumes specified in the `docker-compose.yml`. I don't see a benefit in committing anything in the output directory and since git doesn't track empty directories, I think output should stay.

Reply:
Makes sense. Thanks